src/main/java/org/opentripplanner/standalone/server/Router.java

```diff
@@ -93,7 +93,11 @@ public void startup() {
         }
 
         /* Create Graph updater modules from JSON config. */
-        GraphUpdaterConfigurator.setupGraph(this.graph, routerConfig.updaterConfig());
+        GraphUpdaterConfigurator.setupGraph(
+            this.graph,
+            routerConfig.updaterConfig(),
+            routerConfig.getBikeShareUpdaterEndpoint()
```

Comment:
Can the `gbfsBikeRentalServiceDirectory ` part of the `UpdaterConfig`?

Reply:
It could, but the reason I decided to separate them was to keep router-config json loading completely separate from the reading of this endpoint.

Reply:
The `routerConfig.getBikeShareUpdaterEndpoint()` is just reading the config, not reading the endpoint. There is a mismatch here witch make people ask? Why do we need to pass in two config objects, when the `UpdaterConfig` exist as the "All the GraphUpdaterConfigurator need to know" role?

Reply:
It depends on whether we want to `UpdaterConfig` to have the role of reading all config from all sources, or the role of reading config from json specifically, as is stated in the javadoc.

Reply:
The `bikeShareUpdaterEndpoint` parameter is in the JSON file.

Reply:
I see what you mean now. That method is reading the config, which is an endpoint, just for the configuration. I can name it something like `getBikeRentalServiceDirectoryUrl`.